From 8aa704481f80e55dce430c0c01d276e8ca13018e Mon Sep 17 00:00:00 2001
From: Konstantin Ozerkov <kozerkov@openvz.org>
Date: Fri, 23 Jan 2009 17:43:33 +0300
Subject: [PATCH] Fix broken permissions for Unix98 pty.

This bug is not very critical because modern software can
automatically choose between legacy pty or Unix98 one.

Signed-off-by: Konstantin Ozerkov <kozerkov@openvz.org>
Signed-off-by: Pavel Emelyanov <xemul@openvz.org>
---
 security/device_cgroup.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/security/device_cgroup.c b/security/device_cgroup.c
index d1da90a..ef9fc6b 100644
--- a/security/device_cgroup.c
+++ b/security/device_cgroup.c
@@ -88,7 +88,7 @@ static int devcgroup_can_attach(struct cgroup_subsys *ss,
 #ifdef CONFIG_VE
 static struct dev_whitelist_item default_whitelist_items[] = {
 	{ ~0,                     ~0, DEV_ALL,  ACC_MKNOD },
-	{ UNIX98_PTY_SLAVE_MAJOR, ~0, DEV_CHAR, ACC_READ | ACC_WRITE },
+	{ UNIX98_PTY_MASTER_MAJOR, ~0, DEV_CHAR, ACC_READ | ACC_WRITE },
 	{ UNIX98_PTY_SLAVE_MAJOR, ~0, DEV_CHAR, ACC_READ | ACC_WRITE },
 	{ PTY_MASTER_MAJOR,       ~0, DEV_CHAR, ACC_READ | ACC_WRITE },
 	{ PTY_SLAVE_MAJOR,        ~0, DEV_CHAR, ACC_READ | ACC_WRITE },
-- 
1.6.0.6

